### PR TITLE
kmeleonbrowser.org.K-Meleon version 75.0

### PR DESCRIPTION
**Pull request by Tesco-epic**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15757)

### CODE REVIEW

**Comment by KaranKad** [comment]
`PackageIdentifier` should be `K-Meleon.K-Meleon`

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.installer.yaml:3]:
```suggestion
PackageIdentifier: K-Meleon.K-Meleon
```

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: K-Meleon.K-Meleon
```

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.yaml:3]:
```suggestion
PackageIdentifier: K-Meleon.K-Meleon
```

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
+PackageVersion: 75.0
+Installers:
+- Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.installer.yaml:6]:
```suggestion
- Architecture: x86
```

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
+PackageVersion: 75.0
+PackageLocale: en-US
+Publisher: kmeleonbrowser.org
+PackageName: K-Meleon
+License: kmeleonbrowser.org
+ShortDescription: K-Meleon Installer
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml:9]:
```suggestion
ShortDescription: K-Meleon is an extremely fast, customizable, lightweight web browser based on the Gecko layout engine developed by Mozilla which is also used by Firefox.
```

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: kmeleonbrowser.org.K-Meleon
+PackageVersion: 75.0
+PackageLocale: en-US
+Publisher: kmeleonbrowser.org
+PackageName: K-Meleon
+License: kmeleonbrowser.org
```
Comment (ItzLevvie) [inline manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.locale.en-US.yaml:8]:
```suggestion
License: GNU General Public License v3
LicenseUrl: https://www.gnu.org/licenses/gpl-3.0.txt
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/k/kmeleonbrowser/org/K-Meleon/75.0/kmeleonbrowser.org.K-Meleon.installer.yaml:3]: You'll also need to update the folder names and the `.yaml` files using GitHub's desktop client.

**Pull request closed without merging**